### PR TITLE
completed the card component

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**
-vertical card
-horizontal card
-card dismiss
-text on card
-text only card
-badge on card

### CODE REVIEW

**Review by ajit-sharmaGH — COMMENTED** [review]
card file in html doc 👉260 to 361

File: CSS/card.css [hunk 1]
```diff
@@ -1,3 +1,148 @@
...
+    margin: 2px 10px;
+    text-align: left;
+    font-weight: bold;
+}
+
+.text-on-card {
+    margin-top: -4rem;
+    margin-left: 1rem;
+    margin-bottom: 2rem;
+}
```
Comment (mariyasada) [inline CSS/card.css:115]: you can use margin: -4rem 0rem 2rem 1rem

**Review by mariyasada — APPROVED** [review]
look at the comment

File: CSS/card.css [hunk 2]
```diff
@@ -1,3 +1,148 @@
...
+    border-radius: 0;
+    margin: 2px 5px 2px 5px;
+    background-color: var(--pink);
+    color: var(--blueviolet);
+    font-size: 1rem;
+    font-family: monospace;
+}
+
+.card-btn:hover {
+    color: hotpink;
```
Comment (t007rushi) [inline CSS/card.css:66]: use color variables

File: CSS/card.css [hunk 3]
```diff
@@ -1,3 +1,148 @@
...
+}
+
+.card-image {
+    height: 35vh;
+    width: 13rem;
+    border: 0.2rem solid var(--white);
+    border-radius: 0.5rem;
+}
+
+.card-badge {
```
Comment (kotalwar-krishna) [inline CSS/card.css:27]: one suggestion is be consistent with unit (use rem)

**Review by t007rushi — CHANGES_REQUESTED** [review]
General suggestion : Add some padding + margin to some components that look & feel good.

**Review by kotalwar-krishna — APPROVED** [review]
be consistent with units. sometimes you use rem. Sometimes you use px or vh or em.

The preview is good.

**Comment by ajit-sharmaGH** [comment]
> General suggestion : Add some padding + margin to some components that look & feel good. 

Thanks for the feedback, will do that

**Comment by ajit-sharmaGH** [comment]
> look at the comment

sure

**Comment by ajit-sharmaGH** [comment]
> be consistent with units. sometimes you use rem. Sometimes you use px or vh or em.
> 
> The preview is good.

okk, will care about that

**Pull request merged**